metrics/size.table/query.go

```diff
@@ -0,0 +1,55 @@
+package sizetable
+
+import (
+	"fmt"
+	"strings"
+)
+
+func TableSizeQuery(set map[string]string) (string, error) {
+	query := "SELECT table_schema AS db, table_name as tbl, COALESCE(data_length + index_length, 0) AS tbl_size_bytes FROM information_schema.TABLES"
+	where := " WHERE "
+	if include := set[OPT_INCLUDE]; include != "" {
+		var include_tables []string
```

Comment:
Go style is camelCase, not snake_case (except, sometimes, for const and test func names).